### ISSUE TITLE
Error connecting to Mongo server context deadline exceeded

### ISSUE THREAD

**Issue by rubenqba** [Kaginari/terraform-provider-mongodb#1]
Hi, I want to use the MongoDB provider but it fails to connect to the server. The error message I got is:

`Error: Error connecting to Mongo server context deadline exceeded`

 I presume it is because in my case I need to use the SSL parameter to make the connection, but I can't be sure. This is how looks a secure MongoDB connection string:

`mongodb://<user>:<password>@<hostname>/<database>?ssl=true&authSource=admin`

I'd like to use this provider as it is just what I need for my project. Here a snippet of code I used from the doc samples. 

```
# Configure the MongoDB Provider
provider "mongodb" {
  host = var.mongodb_host
  port = var.mongodb_port
  username = var.mongodb_username
  password = var.mongodb_password
  auth_database = var.mongodb_authdb
}

resource "mongodb_db_role" "example_role" {
  name = "role_name"
  database = "my_database"
  privilege {
    db = "admin"
    collection = "*"
    actions = ["collStats"]
  }
}
```

**Comment by ITMonta** [comment]
@rubenqba Thank you , i submited the issue  will look into that for a fix

**Comment by ITMonta** [comment]
Fixed in version v0.0.3
https://registry.terraform.io/providers/Kaginari/mongodb/latest/docs

**Comment by VladyslavLeveniukSecurrency** [comment]
I have the same issue on version 0.1.5
